docs/production/authentication-methods.md

```diff
@@ -698,11 +700,28 @@ another IdP.
    /home/zulip/deployments/current/manage.py logout_all_users
    ```
 
+#### SP-initiated Single Logout
+
+After configuring IdP-initiated Logout, you only need to set
+`SAML_ENABLE_SP_INITIATED_SINGLE_LOGOUT = True` in
+`/etc/zulip/settings.py` to also enable SP-initiated Logout. When this
+is active, a user who logged in to Zulip via SAML, upon clicking
+"Logout" in the Zulip web app will be redirected to the IdP's Single
+Logout endpoint with a `LogoutRequest`. If a successful
+`LogoutResponse` is received back, their current Zulip session will be
+terminated.
+
+Note that this doesn't work in the case of desktop and mobile application
+and is reserved to the browser.
```

Comment:
Does this documentation need to be updated?

Reply:
This is tweaked in ` saml: Make SP-initiated SLO work in the desktop application.` to say:
```
Note that this doesn't work when logging out of the mobile application
since the app doesn't use sessions and relies on just having the user's
API key.
```